src/plugins/routing.js

```diff
@@ -27,23 +27,42 @@ function localePathFactory (i18nPath, routerPath) {
       route = { name: route }
     }
 
-    // Build localized route options
-    let name = route.name + (STRATEGY === STRATEGIES.NO_PREFIX ? '' : routesNameSeparator + locale)
+    let localizedRoute
+    const router = this[routerPath]
```

Comment:
nit: This line moved up here but we don't use it until later so I would prefer its original location. :)